specification/storage/resource-manager/Microsoft.Storage/stable/2021-06-01/examples/StorageAccountCreateWithImmutabilityPolicy.json

```diff
@@ -0,0 +1,70 @@
+{
```

Comment:
As VLW is also available in update account , please also add update account sample.

Reply:
done